.github/workflows/unit_tests.yml

```diff
@@ -72,92 +75,135 @@ jobs:
-        SLACK_TITLE: "Workflow `${{ github.workflow }}`: ${{ job.status }}"
-        SLACK_MESSAGE: "Run <https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|#${{ github.run_number }}> on <https://github.com/${{ github.repository }}/|${{ github.repository }}@${{ github.ref }}>"
-        SLACK_FOOTER: "<https://github.com/${{ github.repository }}/commit/${{ github.sha }}|View commit>"
-        MSG_MINIMAL: true # use compact slack message format
+      # Set the value of DJANGO_DB_BACKEND which is used in ci/testsettings.py to
+      # configure django's ORM based on whether we're testing postgres or mysql
+      - name: Set django database backend adapter
+        env:
+          BACKEND: ${{ matrix.db }}
+        run: echo "DJANGO_DB_BACKEND=$(echo "$BACKEND")" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # use docker cp to copy the configset, then bash to set ownership to solr
+      - name: Copy Solr configset to solr service
+        run: |
+          docker cp solr_conf ${{ job.services.solr.id }}:/opt/solr/server/solr/configsets/sandco
+          docker exec --user root ${{ job.services.solr.id }} /bin/bash -c "chown -R solr:solr /opt/solr/server/solr/configsets/sandco"
+
+      - name: Copy solr configsets to solr home directory (Solr 8 only)
+        run: "docker exec -d ${{ job.services.solr.id }} cp -r /opt/solr/server/solr/configsets /var/solr/data"
+        if: ${{ matrix.solr == 8.6 }}
+
+      # NOTE: for duration of migration to PUL, use python version from build matrix
+      # Python version to use is stored in the .python-version file, which is the
+      # convention for pyenv: https://github.com/pyenv/pyenv
+      # - name: Get Python version
+      #   run: echo "PYTHON_VERSION=$(cat .python-version)" >> $GITHUB_ENV
+
+      # use python version for current build matrix
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-${{ matrix.python }}-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            pip-${{ matrix.python }}-${{ hashFiles('requirements.txt') }}
+            pip-${{ matrix.python }}
+            pip-
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install -r dev-requirements.txt
+
+      # For mysql only, grant permission to the DB_USER to create databases so
+      # that it doesn't fail trying to create the django test database
+      # - name: Grant permissions to mysql user
+      #   if: matrix.db == 'mysql'
+      #   env:
+      #     GRANT_CMD: "grant all privileges on *.* to '${{ env.DB_USER }}'@'%';"
+      #   run: echo "$GRANT_CMD" | mysql -h 127.0.0.1 --user=root --password="$DB_PASSWORD"
```

Comment:
no longer needed, i guess?

Reply:
I couldn't get it to work this way and reverted back to using the mysql root user that the build was using before I added the matrix. I'll go ahead and remove to clean it up